src/widgets/hits/hits.tsx

```diff
@@ -52,6 +53,10 @@ const renderer = ({
       results={results}
       templateProps={renderState.templateProps}
       insights={insights as InsightsClientWrapper}
+      sendEvent={(event: InsightsEvent) => {
```

Comment:
The function reference is different every time. Is it OK in our case?

Reply:
It's okay because this `sendEvent` is used [in the Preact context](https://github.com/algolia/instantsearch.js/blob/feat/send-events/src/lib/insights/listener.tsx#L76:L76).